packages/fxa-settings/src/components/Settings/DataCollection/index.tsx

```diff
@@ -53,7 +53,12 @@ export const DataCollection = () => {
   }, [account, alertBar, l10n, setIsSubmitting]);
 
   return (
-    <section className="mt-11" data-testid="settings-data-collection">
+    <section
+      className="mt-11"
+      data-testid="settings-data-collection"
+      id="data-collection-section"
```

Comment:
Q: any concerns with this having a `data-` prefix (specifically regarding [data attributes](https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes))?

Reply:
We are using `data-testid` throughout the codebase, this is unchanged here. I would love for us to stop using them though in favor of more accessible references for tests.